tests/pkg/webhook/pods.go

```diff
@@ -37,6 +37,12 @@ func (wh *webhook) admitPods(ar v1beta1.AdmissionReview) *v1beta1.AdmissionRespo
 	reviewResponse := v1beta1.AdmissionResponse{}
 	reviewResponse.Allowed = false
 
+	if !wh.namespaces.Has(namespace) {
```

Comment:
when another spec is in deleting the namespace, this webhook will reject the DeleteCollection request, we can simply ignore requests in other namespaces because each test spec runs in an isolated namespace now